sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -325,6 +325,18 @@ public partial class SimpleFieldAttribute : System.Attribute
 }
 namespace Azure.Search.Documents.Indexes.Models
 {
+    public partial class AmlSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
```

Comment:
`Aml` isn't a great prefix to most of the world.  Could we call this something liker `MachineLearningSkill` or maybe `AzureMachineLearningSkill`?